gattc_windows.go

```diff
@@ -360,14 +361,29 @@ func (c DeviceCharacteristic) Read(data []byte) (int, error) {
 	return len(readBuffer), nil
 }
 
+// EnableNotifications enables notifications in the Client Characteristic
+// Configuration Descriptor (CCCD). Just the Notify mode is supported.
+func (c DeviceCharacteristic) EnableNotifications(callback func(buf []byte)) error {
+	err := c.EnableNotificationsWithMode(false, callback)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
 // EnableNotifications enables notifications in the Client Characteristic
 // Configuration Descriptor (CCCD). This means that most peripherals will send a
 // notification with a new value every time the value of the characteristic
 // changes.
-func (c DeviceCharacteristic) EnableNotifications(callback func(buf []byte)) error {
-	if (c.properties&genericattributeprofile.GattCharacteristicPropertiesNotify == 0) &&
-		(c.properties&genericattributeprofile.GattCharacteristicPropertiesIndicate == 0) {
-		return errNoNotify
+func (c DeviceCharacteristic) EnableNotificationsWithMode(usingIndicate bool, callback func(buf []byte)) error {
```

Comment:
I'm not a big fan of using boolean flags for this. It may make sense for methods that enable or disable something. But I think that in this case it is confusing as the user needs to know what `true` or `false` means.

An enum may be a better option. Maybe something like this?
```go
type NotificationMode = genericattributeprofile.GattCharacteristicProperties

const (
	NotificationModeNotify   NotificationMode = genericattributeprofile.GattCharacteristicPropertiesNotify
	NotificationModeIndicate NotificationMode = genericattributeprofile.GattCharacteristicPropertiesIndicate
)
```

This also allows simplifying the branching inside the method:
```go
func (c DeviceCharacteristic) EnableNotificationsWithMode(mode NotifyMode, callback func(buf []byte)) error {
	if mode != NotificationModeIndicate && mode != NotificationModeNotify {
		return errInvalidNotificationMode
	}
	if c.properties&mode == 0 {
		return errNoNotify
	}
	...
}
```